Sources/SourceKitLSP/LanguageService.swift

```diff
@@ -76,6 +76,18 @@ public struct RenameLocation: Sendable {
   let usage: Usage
 }
 
+/// The textual output of a module interface.
+public struct GeneratedInterfaceDetails: ResponseType, Hashable {
+
```

Comment:
Since you’re touching this anyway, could you remove the superfluous newline?

Reply:
Done! Please have a look :)